### PR TITLE
docs: add login and user mode docs

### PR DESCRIPTION
**Pull request by pyphilia**
Resolve issue #218 and #217 with docs describing our decisions for the user login and the user modes.

### CODE REVIEW

File: docs/authentication/authenticaton-proposals.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# User Authentication
+
+Since Graasp Desktop v.0.15.4, a user authentication is available in order for multiple users to use the application.
+
+![Login Screen](./img/loginScreen.png)
+
+The authentication process is triggered right at the launch of the application. It asks for a username before allowing access to the application. Each new username will create a corresponding user account on the fly. If a username already exists in the database, the user will sign in with the corresponding account and be able to access previously stored data.
+
+One can use the application anonymously without restriction, but any saved data will not be retrievable after the user closes the
+session.
```
Comment (juancarlosfarah) [inline docs/authentication/authenticaton-proposals.md:10]: Remove line break here, no?

File: docs/authentication/userMode.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+and researchers can use it to conduct their studies. They require specific applications
+to track the users, as well as a mechanism to manage consent from users to have legal
+access to their data.
+While Developers and Researchers also have their importance in the development of
+Graasp Desktop and could have their own modes, as part of this thesis, we only focus
+on teacher and student differences in order to develop a teacher mode.
+
+|           Action            | Student  |             Teacher              |
+| :-------------------------: | :------: | :------------------------------: |
+|        Visit a Space        |   yes    |               yes                |
```
Comment (juancarlosfarah) [inline docs/authentication/userMode.md:47]: Capitalisation consistency

**Review by juancarlosfarah — CHANGES_REQUESTED** [review]
Thanks for this! Only minor changes. Can merge after.

**Comment by juancarlosfarah** [comment]
@hasanagh, this PR will be useful for you to see.

**Comment by hasanagh** [comment]
> @hasanagh, this PR will be useful for you to see.

Thanks! I checked it!

**Pull request merged**